lib/Terminal/Print.pm6

```diff
@@ -76,20 +76,15 @@ class Terminal::Print {
     subset Valid::Y of Int is export where * < rows();
     subset Valid::Char of Str is export where *.chars == 1;
 
-    has Terminal::Print::CursorProfile $.cursor-profile;
+    has Terminal::Print::CursorProfile $.cursor-profile = 'ansi';
     has $.move-cursor;
 
-    method new( :$cursor-profile = 'ansi' ) {
-        my $columns      = columns();
-        my $rows         = rows();
-        my $move-cursor  = move-cursor-template($cursor-profile);
-        my $current-grid = Terminal::Print::Grid.new( $columns, $rows, :$move-cursor );
-        self.bless( :$columns, :$rows, :$current-grid,
-                    :$cursor-profile,  :$move-cursor );
-    }
+    submethod TWEAK(|) {
+        self.setup;
 
-    submethod BUILD( :$!current-grid, :$!columns, :$!rows, :$!cursor-profile, :$!move-cursor ) {
-        push @!grids, $!current-grid;
+        unless PROCESS::<$TERMINAL>:exists {
+            PROCESS::<$TERMINAL> = self;
+        }
```

Comment:
PROCESS::<$TERMINAL> //= self;

Reply:
I wouldn't overwrite a symbol no matter what it contains. It might be like this on purpose and set by something else.